src/cloud-api-adaptor/install/overlays/gcp/kustomization.yaml

```diff
@@ -31,6 +31,7 @@ configMapGenerator:
   #- ROOT_VOLUME_SIZE="10" # Uncomment and set if you want to use a specific root volume size. Defaults to 10
   #- TUNNEL_TYPE="" # Uncomment and set if you want to use a specific tunnel type. Defaults to vxlan
   #- VXLAN_PORT="" # Uncomment and set if you want to use a specific vxlan port. Defaults to 4789
+  #- TAGS="" # Uncomment and add key1=value1,key2=value2 etc if you want to use specific tags for podvm
```

Comment:
Hi @beraldoleal !

I suggest a comment similar to "Tags must already exist in the GCP project" here too.